installing_cloudforms_on_red_hat_enterprise_virtualization/changes.adoc

```diff
@@ -0,0 +1,42 @@
+
+************************************* file: configuring_cloudform.adoc ************************************
+line: 73 rephrased from:
+
+======
+Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, Red Hat recommends adding the disk before starting the appliance. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
+======
+
+
+To:
+
+======
+Before installing an internal database, add a disk to the infrastructure hosting your appliance. See the documentation specific to your infrastructure for instructions on how to add a disk. As a storage disk usually cannot be added while a virtual machine is running, adding the disk before starting the appliance is recommended. {product-title} only supports installing of an internal VMDB on blank disks. The installation will fail if the disks are not blank.
+======
+
+
+Rephrased line 108 form:
+
+The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, Red Hat recommends that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
+
+
+To:
+
+The `postgresql.conf` file used with {product-title} databases requires specific settings for correct operation. For example, it must correctly reclaim table space, control session timeouts, and format the PostgreSQL server log for improved system support. Due to these requirements, it is recommended that external {product-title} databases use a `postgresql.conf` file based on the standard file used by the {product-title} appliance.
+**********************************************************************************************************
+
+
+********************************** file: installing_cloudforms.adoc **************************************
+Issue on line 17:
+
+. From the list of installers and images, select the *CFME Red Hat Virtual Appliance* download link.
+
```

Comment:
What do you mean by RHEV being already done? We were sent a .doc with a table with needed changes in the documentation. Are you talking about the documentation or the development project?